src/components/form/form.tsx

```diff
@@ -0,0 +1,32 @@
+import { FieldValues, FormProvider, useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { HookFormProps } from "./form.types.ts";
+import { Form as AntdForm } from "antd";
+
+const Form = <T extends FieldValues>({
+    schema,
+    onSubmit,
+    children,
+    defaultValues,
+    ...props
+}: HookFormProps<T>) => {
```

Comment:
```suggestion
}: FormProps<T>) => {
```
```suggestion
}: HookFormProps<T>) => {
```
Props should be named like component or just Props